Mathlib/Order/Filter/Germ.lean

```diff
@@ -239,76 +238,77 @@ theorem coe_compTendsto' (f : α → β) {lc : Filter γ} {g : Germ lc α} (hg :
   rfl
 #align filter.germ.coe_comp_tendsto' Filter.Germ.coe_compTendsto'
 
-/-- Given a germ `f : germ l β` and a function `g : γ → α`, where `l : filter α`, if `g` tends
-to `l` along `lc : filter γ`, then the composition `f ∘ g` is well-defined as a germ at `lc`. -/
+/-- Given a germ `f : Germ l β` and a function `g : γ → α`, where `l : Filter α`, if `g` tends
+to `l` along `lc : Filter γ`, then the composition `f ∘ g` is well-defined as a germ at `lc`. -/
 def compTendsto (f : Germ l β) {lc : Filter γ} (g : γ → α) (hg : Tendsto g lc l) : Germ lc β :=
   f.compTendsto' _ hg.germ_tendsto
 #align filter.germ.comp_tendsto Filter.Germ.compTendsto
 
 @[simp]
 theorem coe_compTendsto (f : α → β) {lc : Filter γ} {g : γ → α} (hg : Tendsto g lc l) :
-    (f : Germ l β).comp_tendsto g hg = f ∘ g :=
+    (f : Germ l β).compTendsto g hg = f ∘ g :=
   rfl
 #align filter.germ.coe_comp_tendsto Filter.Germ.coe_compTendsto
 
-@[simp]
+-- @[simp] -- Porting note: simp cannot prove this
```

Comment:
If simp cannot prove this, please don't comment out the `@[simp]` attribute.
Instead, if the linter complains, add `nolint simpNF`.
```suggestion
@[simp, nolint simpNF] -- Porting note: simp cannot prove this
```